app/controllers/provider_interface/conditions_controller.rb

```diff
@@ -5,13 +5,23 @@ class ConditionsController < ProviderInterfaceController
     before_action :requires_make_decisions_permission
 
     def edit
-      @form_object = ConfirmConditionsForm.new
+      if FeatureFlag.active?(:individual_offer_conditions)
+        @form_object = ConfirmConditionsWizard.new(condition_statuses_store, offer: @application_choice.offer)
+        @form_object.save_state!
+      else
+        @form_object = ConfirmConditionsForm.new
+      end
```

Comment:
Not sure about the usage of the feature flag in this way. I think it might be more comfortable if we add new routes and a controller. That way we don't mix flows together.

WDYT?

Reply:
Adding a separate controller